tests/extra/job_management/test_job_splitting.py

```diff
@@ -0,0 +1,242 @@
+import pytest
+import shapely
+
+from openeo.extra.job_management.job_splitting import (
+    BoundingBox,
+    JobSplittingFailure,
+    SizeBasedTileGrid,
+    reproject_bounding_box,
+    split_area,
+)
+
+
+@pytest.fixture
+def mock_polygon_wgs():
+    return shapely.geometry.box(0.0, 0.0, 1.0, 1.0)
+
+
+@pytest.fixture
+def mock_polygon_utm():
+    return shapely.geometry.box(0.0, 0.0, 100_000.0, 100_000.0)
+
+
+@pytest.fixture
+def mock_dict_no_crs():
+    return {
+        "west": 0.0,
+        "south": 0.0,
+        "east": 1.0,
+        "north": 1.0,
+    }
+
+
+@pytest.fixture
+def mock_dict_with_crs_utm():
+    return {
+        "west": 0.0,
+        "south": 0.0,
+        "east": 100_000.0,
+        "north": 100_000.0,
+        "crs": "EPSG:3857",
+    }
```

Comment:
I guess it's a matter of taste, but these four fixtures `mock_polygon_wgs`, ... `mock_dict_with_crs_utm` feel like overkill to me.
I guess it is to be DRY, but in testing I think this extra level of indirection for these simple constructs is worse for readability than violating DRY.
Also, if you still to be DRY, I think you should just use constants instead of pytest fixtures (you are not really using them as fixtures, in terms of setup/teardown)